api/src/routes/public/user.ts

```diff
@@ -189,6 +189,8 @@ export const userPublicGetRoutes: FastifyPluginCallbackTypebox = (
 
         const returnedUser = {
           ...removeNulls(publicUser),
+          // Ensure tri-state preference is preserved even if null
+          sendQuincyEmail: user.sendQuincyEmail,
```

Comment:
At line 144, we explicitly omit this from the public data. I am not sure we want this here.